Makefile

```diff
@@ -44,7 +44,7 @@ mock_gen:
 
 .PHONY: test
 test:
-	GO111MODULE=on go test ./...
+	GO111MODULE=on go test -v -race ./...
```

Comment:
This increases the time to run the tests by quite a bit. Thoughts on adding a new CI job/command for checking races and keeping this command at just `go test ./...`?

Reply:
"time make test" with these setting on my laptop (which isn't very powerful) reports :-
real	1m23.557s
user	3m31.509s
sys	0m25.549s

If I remove the verbose flag, this is reduced to :-
real	0m25.310s
user	0m35.678s
sys	0m9.804s

Assuming this is fast enough?

Reply:
removing the verbose flag is good enough. Thank you!